modules/manually-create-identity-access-management.adoc

```diff
@@ -228,6 +228,24 @@ spec:
 +
 endif::ash[] 
 
+.Sample `Secret` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-image-registry
+  name: installer-cloud-credentials
+data:
+  azure_subscription_id: <base64-encrypted-azure-subscription-id>
```

Comment:
These replaceable values should be using underscores instead of dashes between them (like the next step, step 7 does). Would you mind fixing that before we merge?

Reply:
Addressed